### ISSUE TITLE
Integer values?

### ISSUE THREAD

**Issue by Kondra007** [coleifer/vedis-python#7]
Do I understand you right, that only supported data types in Vedis are strings? 
When trying to save data as 
```python
with Vedis("mydb.vdb") as db:
    db["foo"] = 5
```
my db becomes 180 megabytes (!!!) big instantly!
In python interactive shell:
```python
db = Vedis("mydb.vdb")
db.set("foo",5)
print(db.get("foo"))
(produces emtpy row in console)
```

**Comment by coleifer** [comment]
That's quite odd, testing locally my database size is only 12Kb after performing the above. Can you provide any additional information?

**Comment by coleifer** [comment]
What version of Vedis and what Python version you running?

**Comment by Kondra007** [comment]
@coleifer Using Vedis 0.4.0 (Installed an hour ago) with Python 3.4 on Ubuntu 14.04
Well, that's really weird, because this problems occurs only on specific app (Telegram bot) and only if value is big integer (8 or 9 digits). This way Vedis DB can grow 95 or 180 megabytes. Converting this integer to `str()` solved the problem.

**Comment by coleifer** [comment]
Ok, I just tried using a small integer. I'll try again in a bit when I'm at my computer again.

**Comment by Kondra007** [comment]
I've checked with `db[str(1234567890)] = 123456789` -> `MemoryError`  
Bigger ints produce `OverflowError`, so I'm starting to understand, why error happened. Why 95 or 180 megabytes? No idea.

Is it possible to store large integers in Vedis? Python itself works fine with them.

**Comment by coleifer** [comment]
I verified the behavior with Python 3.5, though it's not present in 2.x. Very odd! I'll look into it, thanks so much for posting.

**Comment by coleifer** [comment]
A-ha! The problem is that when you call `bytes(12345)` in py2 it creates a string representation. In py3 it creates a bytestring of length 12345.

**Comment by coleifer** [comment]
This is fixed and I've pushed 0.4.1. Thanks again. I've also fixed my other cython db bindings `encode()` function.
